proto/base/types/gfspserver/manage.proto

```diff
@@ -146,6 +146,46 @@ message GfSpResetRecoveryFailedListResponse {
   repeated string recovery_failed_list = 1;
 }
 
+message GfSpTriggerRecoverForSuccessorSPRequest {
+  uint32 vgf_id = 1;
+  uint32 gvg_id = 2;
+  int32 replicate_index = 3;
```

Comment:
redundancyIndex maybe more better. Try to keep the naming and semantics consistent.

Reply:
redundancyIndex does not comply with the proto lint standard